tron/core/job.py

```diff
@@ -435,32 +456,37 @@ class JobCollection(object):
     """A collection of jobs."""
 
     def __init__(self):
-        self.jobs = collections.MappingCollection('jobs')
+        self.jobs = {}
```

Comment:
There are other references to self.jobs in this class, do you need to update those too?

Reply:
i've checked - they all use dict-interface, so we're fine